WordPress/UITestsFoundation/Globals.swift

```diff
@@ -31,6 +31,17 @@ public func pullToRefresh(app: XCUIApplication = XCUIApplication()) {
     top.press(forDuration: 0.01, thenDragTo: bottom)
 }
 
+public func waitAndTap( _ element: XCUIElement) {
+    var retries = 0
+    let maxRetries = 10
+    if element.waitForIsHittable(timeout: 10) {
+        while element.isHittable && retries < maxRetries {
+            element.tap()
+            retries += 1
+        }
+    }
+}
```

Comment:
Nice to move this into the UITestsFoundation framework 😄 

While we're at it, it might be nice to update the signature and implementation.

- Inject `maxRetries` and `timeout`
- Use `guard`

```suggestion
public func waitAndTap( _ element: XCUIElement, maxRetries: Int = 10, timeout: TimeInterval = 10) {
    guard element.waitForIsHittable(timeout: timeout) else {
      return
    }
    
    var retries = 0
    while element.isHittable && retries < maxRetries {
            element.tap()
            retries += 1
        }
    }
}
```

Reply:
Out of scope for this PR, but I wonder if we should fail or `throw` if [`waitForIsHittable`](https://github.com/Automattic/XCUITestHelpers/blob/5179cb69d58b90761cc713bdee7740c4889d3295/Sources/XCUITestHelpers/XCUIElement%2BAsync.swift#L5-L16) reaches its timeout.

Reply:
Thanks for looking into it, @mokagio! 
I'll merge it as is and will work on refactoring `waitAndTap( _ element: XCUIElement)` in a new PR. 

> Out of scope for this PR, but I wonder if we should fail or throw if [waitForIsHittable](https://github.com/Automattic/XCUITestHelpers/blob/5179cb69d58b90761cc713bdee7740c4889d3295/Sources/XCUITestHelpers/XCUIElement%2BAsync.swift#L5-L16) reaches its timeout.

Yeah, it makes sense. At a first sight I believe we might want to fail after a timeout.

Reply:
Sounds good to me 😄